src/app/tabs/search/reducer.js

```diff
@@ -71,6 +77,29 @@ const searchReducer = (state = initialState, action) => {
         ...initialState,
       };
     }
+    case VALIDATION_MESSAGE: {
```

Comment:
It would be nice if this had an action name like
```suggestion
    case SET_VALIDATION_MESSAGE: {
```